homeassistant/components/shelly/climate.py

```diff
@@ -160,6 +161,9 @@ def __init__(
             ]
         elif entry is not None:
             self._unique_id = entry.unique_id
+        self._attr_device_info = DeviceInfo(
+            connections={(CONNECTION_NETWORK_MAC, self.coordinator.mac)},
```

Comment:
```suggestion
            connections={(CONNECTION_NETWORK_MAC, coordinator.mac)},
```
If you feel extra fancy, there are some more `self.` that can be removed in the lines above